recipes/satmad/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "satmad" %}
+{% set version = "0.0.3.post1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 113a5117c0b99d9c73677eec9deda29d2d9373dabe0fa446f745da2181e5617c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - flit-core
+    - pip
+    - python
+
+  run:
+    - astropy
+    - matplotlib
+    - numpy
+    - portion
+    - python
+    - scipy
+    - sgp4
+
+test:
+  imports:
+    - satmad
+    - satmad.coordinates
+    - satmad.coordinates.tests
+    - satmad.propagation
+    - satmad.propagation.tests
+    - satmad.tests
+    - satmad.utils
+    - satmad.utils.tests
```

Can we try to add ``pip check`` here please?
```suggestion
    - satmad.utils.tests
  requires:
    - pip
  commands:
    - pip check
```